src/renderer/components/pages/FilterPage.tsx

```
@@ -0,0 +1,373 @@
+        </p>
+
+        {!hasLibrary && (
+          <p className="text-sm text-te-grey-400 italic">Load a library first.</p>
+        )}
+
+        {hasLibrary && (
+          <div className="space-y-te-md">
+
+            {/* Filter rules */}
+            <div>
+              <label className="block text-xs font-medium text-te-grey-600 mb-2 uppercase">Filters</label>
```

ESLint

[error] A form label must be associated with a control.

(jsx-a11y/label-has-associated-control)